server/webhook_http.go

```diff
@@ -29,6 +29,7 @@ var eventParamMasks = map[string]StringSet{
 	"updated_description": NewStringSet(eventUpdatedDescription), // issue description edited
 	"updated_labels":      NewStringSet(eventUpdatedLabels),      // updated labels
 	"updated_prioity":     NewStringSet(eventUpdatedPriority),    // changes in priority
+	"updated_priority":    NewStringSet(eventUpdatedPriority),    // changes in priority
```

Comment:
Probably want to include a comment on the line above, saying we're keeping it around for backwards compatibility

Reply:
@mickmister Added the comment